helix-view/src/editor.rs

```diff
@@ -271,6 +272,8 @@ pub struct Breakpoint {
     pub log_message: Option<String>,
 }
 
+pub trait DynAccessDebug<T>: DynAccess<T> + std::fmt::Debug {}
```

Comment:
?

Reply:
😳 was trying stuff out and forgot to remove that